Dev-Corner/slides/bioblend_api.html

```diff
@@ -0,0 +1,806 @@
+<!DOCTYPE html>
```

Comment:
Can you put on the top:

```
---
layout: tutorial_slides
topic_name: "Dev-Corner"
tutorial_name: bioblend_api
logo: "GTN"
---
```

It is useful to find the good templates.  Like this, you do not need any html in the top. 

Thanks